### PR TITLE
Update Thorma90 Network mainfest and images

### PR DESCRIPTION
**Pull request by EinfachFloTV**

### CODE REVIEW

**Comment by maaaathis** [comment]
You don't need to open a new PR for every change. As long as you commit an change to this branch (https://github.com/EinfachFloTV/server-media/tree/master), this PR is updated automatically.

**Comment by EinfachFloTV** [comment]
Ok sry can you merge?

**Comment by EinfachFloTV** [comment]
Its really fine

**Comment by HyperGaming99** [comment]
Pls Merge

**Comment by EinfachFloTV** [comment]
Can you please merge it? I apologize for creating a second PR.

File: minecraft_servers/thorma90/background.png
Comment (maaaathis) [inline minecraft_servers/thorma90/background.png:1]: Compress

File: minecraft_servers/thorma90/background@2x.png
Comment (maaaathis) [inline minecraft_servers/thorma90/background@2x.png:1]: Compress

File: minecraft_servers/thorma90/manifest.json [hunk 1]
```diff
@@ -2,20 +2,36 @@
     "server_name": "thorma90",
     "nice_name": "Thorma90 Netzwerk",
     "direct_ip": "thorma90.net",
+    "server_wildcards": [
+      "play.thorma90.net",
+      "mc.thorma90.net"
```
Comment (maaaathis) [inline minecraft_servers/thorma90/manifest.json:7]: Non-valid wildcards

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/thorma90/manifest.json (hunk 1 above)
Comment (EinfachFloTV) [reply minecraft_servers/thorma90/manifest.json:7]: it works?

Comment (EinfachFloTV) [reply minecraft_servers/thorma90/manifest.json:7]: its a ailases for minecraft

Comment (maaaathis) [reply minecraft_servers/thorma90/manifest.json:7]: Definition of a DNS wildcard: https://en.wikipedia.org/wiki/Wildcard_DNS_record
These are simple subdomains. This field is intended only for real wildcards that allow every subdomain.

Comment (EinfachFloTV) [reply minecraft_servers/thorma90/manifest.json]: fixed

**Review by DoJapHD — APPROVED** [review]

**Comment by Eptiiix** [comment]
Hallo die Herren,

habt ihr euch den Commit vor dem approven überhaupt mal angeschaut?
@EinfachFloTV hat keinerlei Rechte an "TheVace" und hat es aus der Repository entfernt.

Bitte diese Änderung rückgängig machen.
Anbei der dazugehörige PR: https://github.com/LabyMod/server-media/pull/646

Viele Grüße,
Eptiiix

**Pull request merged**